backend/src/resources/setups.ts

```diff
@@ -0,0 +1,303 @@
+  { setup: "What never fails to liven up the party?", type: SetupType.pickOne },
+  {
+    setup: "IF you like __________________, YOU MIGHT BE A REDNECK.",
+    type: SetupType.pickOne,
+  },
+  { setup: "Make a haiku.", type: SetupType.drawTwoPickThree },
+  { setup: "What made my first kiss so awkward?", type: SetupType.pickOne },
+  {
+    setup:
+      "Hey guys, welcome to Chili's! Would you like to start the night off right with _________________?",
+    type: SetupType.pickOne,
+  },
+  {
+    setup: "I got 99 problems but _______________ ain't one.",
+    type: SetupType.pickOne,
+  },
+  { setup: "__________________. It's a trap!", type: SetupType.pickOne },
+  {
+    setup:
+      "Bravo's new reality show features eight washed-up celebrities living with _____________.",
+    type: SetupType.pickOne,
+  },
+  {
+    setup: "What would grandma find disturbing, yet oddly charming?",
+    type: SetupType.pickOne,
+  },
+  { setup: "__________________. That was so metal.", type: SetupType.pickOne },
+  {
+    setup:
+      "I never truly understood __________________ until I encountered _________________.",
+    type: SetupType.pickTwo,
+  },
+  {
+    setup: "During sex, I like to think about __________________.",
+    type: SetupType.pickOne,
+  },
+  { setup: "What ended my last relationship?", type: SetupType.pickOne },
+  { setup: "What's that sound?", type: SetupType.pickOne },
+  {
+    setup:
+      "Uh, hey guys, I know this was my idea, but I'm having serious doubts about ____________.",
+    type: SetupType.pickOne,
+  },
+  { setup: "Why am I sticky?", type: SetupType.pickOne },
+  {
+    setup:
+      "I'm no doctor but I'm pretty sure what you're suffering from is called \"__________.\"",
+    type: SetupType.pickOne,
+  },
+  { setup: "What's there a ton of in heaven?", type: SetupType.pickOne },
+  {
+    setup:
+      "After four platinum albums and three Grammys, it's time to get back to my roots, to what inspired me to make music in the first place: __________.",
+    type: SetupType.pickOne,
+  },
+  { setup: "What will always get you laid?", type: SetupType.pickOne },
+  {
+    setup:
+      "They said we were crazy. They said we couldn't put __________________ inside of _________. They were wrong.",
+    type: SetupType.pickTwo,
+  },
+  {
+    setup:
+      'Lifetime® presents "_________________: the Story of _________________."',
+    type: SetupType.pickTwo,
+  },
+  {
+    setup: "__________________: kid-tested, mother-approved.",
+    type: SetupType.pickOne,
+  },
+  { setup: "Why can't I sleep at night?", type: SetupType.pickOne },
+  { setup: "What's that smell?", type: SetupType.pickOne },
+  {
+    setup:
+      "After eight years in the White House, how is Obama finally letting loose?",
+    type: SetupType.pickOne,
+  },
+  {
+    setup:
+      "This is the way the world ends This is the way the world ends Not with a bang but with __________________.",
```

Comment:
How do we handle line breaks in setups?

Reply:
This is a weird setup based on a poem - I've fixed formatting in 2a01133 by adding full stops